### ISSUE TITLE
Unable to Generate Builds Based on Item Traits

### ISSUE THREAD

**Issue by Spooky-Noodle** [Pfhoenix/DDONamedGearPlanner#3]
Hey, decided to move the issue here so that I don't clog up the forums. 

I put in a series of filters, with the intention that I would click the 'Start Build' button, and the program would generate an ideal set of items for me to go after. (If I'm misunderstanding how the program is supposed to function, let me know.) 

A screenshot of the build filters is here: 

https://i.imgur.com/1v8U1ef.png

Here's a screenshot of the main UI: 

https://i.imgur.com/WPsgJv6.png

Here is a gif of what happens when I click 'Start Build':

https://gfycat.com/tightglumglobefish

So the dialogue box pops up for a fraction of a second, then disappears, and no items go into the respective slots. 

I can save these sets of filters, and adjust them as necessary, so let me know anything I can do to help identify the problem.

**Comment by Pfhoenix** [comment]
Thanks for the screenshots. I'm in the middle of redesigning the build process, so once I'm done with that, I can setup the filters as you screenshotted and use them as a test case.

**Comment by Spooky-Noodle** [comment]
I did some additional testing with various item filters, and particularly using Minimum Level as a filter, and I think what's really causing problems is just the sheer volume. I was able to take the same item filters from up above, but shrink the ML so that it's only a range of 1-2 levels, and the window successfully generated, began generating builds, but there were 73 trillion combinations. 

So I think that, with the enormous amount of filters that I'm putting in, there's just so many combinations that my computer can't handle all the possibilities. So, I guess to fix it, you'd have to redesign the entire approach to how the program generates builds, which I would normally assume way too complicated, but you said you're "in the middle of redesigning the build process," so I guess not.
